app/managers/DialcodeManager.java

```diff
@@ -272,7 +272,9 @@ private Response validateInput(String metaData) throws Exception {
                     ResponseCode.CLIENT_ERROR);
         }
 
-        String contextSchemaPath = schemaCache.getSchemaPath(jsonldType);
+        if(!jsonldType.equalsIgnoreCase("sb")) validateContextVocabulary(contextJson);
```

Comment:
What is "sb" here.  If you expecting the vocaludation key name of Sunbird, this can be any name.
The key name they can define as per thier naming standards.